@AH-64D Apache Longbow/Addons/fza_ah64_fire/functions/fn_damageEnginefire.sqf

```diff
@@ -0,0 +1,113 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fire_fnc_damageEngineFire
+
+Description:
+    damageEngineFire
+
+Parameters:
+    _heli - The helicopter to modify
+    _eng - the engine that has been assigned to burn
+
+Returns:
+    Nothing
+
+Examples:
+    --- Code
+    [_heli, "right"] spawn fza_fire_fnc_damageEngineFire
+    ---
+
+Author:
+    Snow(dryden), Mattysmith22
+---------------------------------------------------------------------------- */
+params ["_heli","_eng"];
+
+_rand = floor random 3;
+if (_rand > 1) exitwith {};
+
+private _side = [];
+private _sidef = [];
+private _mag = "";
+private _audio1 = "";
+switch _eng do {
+    case "right": {
+        _side = [1.2, -0.8, -1.25];
+        _sidef = [1.2, -0.6, -1.25];
+        _mag = "fza_ah64_e2_fire";
+    };
+    case "left": {
+        _side = [-1, -0.8, -1.25];
+        _sidef = [-1, -0.6, -1.25];
+        _mag = "fza_ah64_e1_fire";
+    };
+    case "apu": {
+        _side = [0, -0.8, -1.25];
+        _sidef = [0, 0.2, -1.25];
+        _mag = "fza_ah64_apu_fire";
+    };
+};
+
+if (_heli getVariable _mag) exitwith {};
+
+_heli setVariable[_mag, true, true];
+["fza_engineFire", [_heli, _eng]] call CBA_fnc_globalEvent;
+
+Private _PrimaryFBAvailable = !(_heli getVariable "fza_ah64_firepdisch");
```

Comment:
This entire file needs capitalisation fixing

`private`

Variables should not start with a capital